examples/distributed-tracing/README.md

```diff
@@ -0,0 +1,25 @@
+# Distributed Tracing Example
+This example uses `docker-compose`, and illustrates the distributed tracing functionality of OpenTelemetry. An HTTP request to service-one will make multiple con current HTTP requests, each of which is injected with a `traceparent` header.
```

Comment:
"con current" -  is this a pun, I don't get?

Reply:
nope, just a misplaced space. I changed it to "asynchronous"